node-src/ui/tasks/verify.ts

```diff
@@ -43,8 +43,8 @@ export const runOnlyNames = (ctx: Context) => ({
     .join(', ')}`,
 });
 
-export const awaitingUpgrades = (ctx: Context, upgradeBuilds: { completedAt?: number }[]) => {
-  const pending = upgradeBuilds!.filter((upgrade) => !upgrade.completedAt).length;
+export const awaitingUpgrades = (_ctx: Context, upgradeBuilds: { completedAt?: number }[]) => {
+  const pending = upgradeBuilds.filter((upgrade) => !upgrade.completedAt).length;
```

Comment:
What happened to the `!` in here and is that a problem?  Also `_ctx` -> `_` maybe?

Reply:
I'm honestly not sure why `!` was there unless the `upgradeBuilds` arg was optional at one point. Since it's a required field, we don't need the non-null assertion there.